src/main/java/cz/smarteon/loxone/Codec.java

```diff
@@ -87,6 +88,26 @@ public static String bytesToHex(byte[] bytes) {
         return sb.toString();
     }
 
+    /**
+     * Encodes given bytes to Base64 and returns it as IS0-8859-1 encoded String.
+     * @param bytes bytes to encode
+     * @return encoded bytes
+     */
+    @NotNull
+    public static String bytesToBase64(final @NotNull byte[] bytes) {
+        return Base64.getEncoder().encodeToString(bytes);
+    }
+
+    /**
+     * Decodes given Base64 encoded String to bytes.
+     * @param base64 to decode
+     * @return decoded String
+     */
```

Comment:
What about to add `@throws IllegalArgumentException`?

Reply:
done